RELEASE_MANAGEMENT.md

```diff
@@ -0,0 +1,140 @@
+
+* Creating and managing a GitHub Milestone for the release.
+* Creating and managing a GitHub Project board for the release.
+* Creating and managing a GitHub Discussion Boards announcement for the release.
+* Working through the [Release Phases](#release-phases) for the release.
+
+This management process ultimately results in the manager of the release
+shipping the release as per the [Release Process](#release-process). We will
+go into more detail about this in the sections that follow.
+
+> **Note**: Patch releases need someone assigned to them as well, however they
+> just happen organically, and as soon as possible. We currently we don't bother
+> sending patch releases through this entire process.
+
+## Assigning a Release Manager
+
+The [maintainers] will find and assign a release managers for upcoming
+releases. A release manager can be a community member on the contributor ladder
+who volunteers (at the discretion of the maintainers), or otherwise the
+responsibility falls back to the maintainers.
+
+[maintainers]:/OWNERS_ALIASES
+
+## GitHub Milestone
+
+Once a manager is assigned for the release, they must create a [GitHub
+Milestone] for the release, where the name of the milestone is the eventual tag
+that will be cut (e.g., `v1.0.0`).
+
+The release manager must mark themselves as the manager for the release at the
+top of the milestone's description, and add any description to the milestone
+that communicates to the community any major themes in the release, and
+important notes:
+
+```console
+release-manager: @<name>
+
+This milestone introduces the following new experimental features:
+
+* <experimental-feature-1>
+* <experimental-feature-2>
+* <experimental-feature-3>
+
+This milestone graduates the following features:
+
+* <GA-feature-1>
+* <GA-feature-2>
+
+feature-freeze: <mm-dd-yy>
+```
+
+> **Note**: It's normal for this description to change over time, as things are
+> never completely certain at the onset.
+
+The release manager must assign a due date for the milestone, to communicate to
+the community the intended date to ship the release. They must also indicate an
+expected [feature freeze](#freeze-phase) date in the description.
+
+> **Note**: This can change over time as well, but it's valuable to try and
+> communicate dates in this way as many implementations and community members
+> plan their releases on ours.
+
+[GitHub Milestone]:https://github.com/kubernetes-sigs/gateway-api/milestones
+
+## GitHub Project Board
+
+The release manager must create a [project board] for the release, which will
+be used to help track the work in the [milestone](#github-milestone) over time
+to refine and analyze progress as the release is ongoing. The board should
+include `Next`, `In Progress`, `Review` and `Done` columns to indicate progress
+(which can be shared via updates on the community syncs as well).
+
+[project board]:https://github.com/kubernetes-sigs/gateway-api/projects
+
+## Release Phases
```

Comment:
This seems like a significant departure from https://gateway-api.sigs.k8s.io/contributing/release-cycle/#phases? Is that intentional?

Reply:
fixed.